Sources/web3swift/Operations/WriteOperation.swift

```diff
@@ -13,23 +13,33 @@ public class WriteOperation: ReadOperation {
     // FIXME: Rewrite this to CodableTransaction
     /// Sends raw transaction for write operation.
     /// - Parameters:
-    ///   - password: Password for private key.
-    ///   - policies: Custom policies for how to resolve (optional). Default is auto.
-    public func writeToChain(password: String, policies: Policies = .auto) async throws -> TransactionSendingResult {
+    ///   - password: Password for the private key in the keystore manager attached to the provider 
+    ///   you set to `web3` passed in the initializer.
+    ///   - policies: Determining the behaviour of how transaction attributes like gas limit and 
+    ///   nonce are resolved. Default value is ``Policies/auto``.
+    ///   - sendRaw: If set to `true` transaction will be signed and sent using `eth_sendRawTransaction`.
+    ///   Otherwise, no signing attempts will take place and the `eth_sendTransaction` RPC will be used instead.
+    ///   Default value is `false`.
+    public func writeToChain(password: String, policies: Policies = .auto, sendRaw: Bool = false) async throws -> TransactionSendingResult {
```

Comment:
I'm still not sure why would we set default value for `sendRaw` to false.
One issue I see clearly now with this implementation - everyone who uses the library will suddenly start sending unsigned transactions to the blockchain and that will basically break everything for those developers.
So I think in the current state this PR won't pass.

Could you please either paste a link here or maybe describe once again why did you decide to set the value to `false`?

Reply:
Probably found it - https://github.com/web3swift-team/web3swift/pull/670/files#r1027680672

So this was the issue?
> yes setting it true by default makes the tests fail

Reply:
yea this can't be the reasoning. pharms would have to investigate why the tests fail then.

I see it like you @JeneaVranceanu we can't just change this now.

Reply:
ok I will investigate the failed tests